lib/tasks/scsb.rake

```diff
@@ -74,5 +74,11 @@ namespace :scsb do
     task full: :environment do
       ScsbImportFullJob.perform_later
     end
+
+    desc "Used for monthly cron job - downloads files for full partner record set only on Saturdays"
+    task full_saturdays_only: :environment do
+      abort "This task will only run on Saturdays, to facilitate monthly cron job. If you want to run this job manually on another day, use scsb:import:full" unless Date.today.saturday?
```

Comment:
My understanding is by having 'unless Date.today.saturday' cron will try to run the job and then abort it. Why to do that since we can set 6 (for Saturday) in the cron job. Between 8-14 there is always a Saturday and cron will run once without a need to abort daily until it's a Saturday.

Reply:
Crontab is weird, and it uses an "OR" for correct day OR correct day of the week, so that would make it run every day in that week, and every Saturday. see https://stackoverflow.com/questions/34357126/why-crontab-uses-or-when-both-day-of-month-and-day-of-week-specified